README.md

```diff
@@ -150,6 +150,12 @@ http_request_duration_seconds_sum{handler="metrics",method="get"} 0.021113919999
 http_request_duration_seconds_count{handler="metrics",method="get"} 30
 ```
 
+kube-state-metrics exposes the build info metric:
```

Comment:
Nit: Would be good to add a few lines what those are useful for. Similiar to have we have them for the list and watch errors above.

Reply:
Fixed.